src/lib/components/ReactRRuleGenerator.js

```diff
@@ -10,7 +10,7 @@ import configureInitialState from '../utils/configureInitialState';
 import '../styles/index.css';
 
 class ReactRRuleGenerator extends Component {
-  state = configureInitialState(this.props.config);
+  state = configureInitialState(this.props.config, this.props.calendarComponent);
```

Comment:
What do you think about having this `calendarComponent` in `props.config` rather than as separate entry?

It could even be scoped to `props.config.end.calendarComponent`